### ISSUE TITLE
IPA ad trust

### ISSUE THREAD

**Issue by Yalafeg** [freeipa/ansible-freeipa#322]
Hello,

Is there a module to add/manage AD trust or to manage override users?

Regards,

**Comment by t-woerner** [comment]
There is a PR for trust: https://github.com/freeipa/ansible-freeipa/pull/205
Is this what you are searching for?

**Comment by Yalafeg** [comment]
Thank you, and what about the option override users? I need to automate the command: "ipa group-add-member admin_external --idoverrideusers=test@example.com". I don't see any option related to this command, are there any?

**Comment by t-woerner** [comment]
No, these options are sadly not supported yet. But the needed changes should be small.

**Comment by Yalafeg** [comment]
thank you

**Comment by rjeffman** [comment]
Hello, @Yalafeg.

I was playing around with iduseroverrides, and did an initial implementation to provide support for `idoverrideusers`, but I'm not able to test it, since I don't have an AD server to create a trust.

If you are willing to test it, it is available an my fork https://github.com/rjeffman/ansible-freeipa, branch `ipagroup_add_idoverrideuser`.

**Comment by rjeffman** [comment]
As of version 1.12.0, the only module missing for managing AD objects stored in FreeIPA is `trustconfig`, and as this issue is about overrides, which are currently supported, I'm closing it.

Thank you for reporting, and if something is not working related to this issue, please reopen it, or create a new, more specific one.
